plugins/ocm/README.md

```diff
@@ -1,104 +1,114 @@
-4. Following `ClusterRole` must be granted to `ServiceAccount` accessing the Hub cluster:
-   ```yaml
-   kind: ClusterRole
-   apiVersion: rbac.authorization.k8s.io/v1
-   metadata:
-     name: backstage-ocm-plugin
-   rules:
-     - apiGroups:
-         - cluster.open-cluster-management.io
-       resources:
-         - managedclusters
-       verbs:
-         - get
-         - watch
-         - list
-     - apiGroups:
-         - internal.open-cluster-management.io
-       resources:
-         - managedclusterinfos
-       verbs:
-         - get
-         - watch
-         - list
-   ```
+The Open Cluster Management (OCM) plugin integrates your Backstage instance with the `MultiClusterHub` and `MultiCluster` engines of OCM.
 
 ## Capabilities
 
-- All clusters represented as `ManagedCluster`s in MultiClusterHub/MCE are discovered and imported into Backstage catalog:
+The OCM plugin has the following capabilities:
+
+- All clusters represented as `ManagedCluster` in `MultiClusterHub` or MCE are discovered and imported into the Backstage catalog, such as:
   - Entity is defined as `kind: Resource` with `spec.type` set to `kubernetes-cluster`.
-  - Links to OCP console, OCM console and OpenShift Cluster Manager are provided in `metadata.links`.
-- Show real time data from OCM on Resource entity page including:
-  - Cluster current status (up, down, etc.)
-  - Cluster details (console link, OCP and Kubernetes version, etc.)
+  - Links to the OpenShift Container Platform (OCP) console, OCM console, and OpenShift Cluster Manager are provided in `metadata.links`.
+- Shows real-time data from OCM on the Resource entity page, including:
+  - Cluster current status (up or down)
+  - Cluster details (console link, OCP, and Kubernetes version)
   - Details about available compute resources on the cluster
 
-## Usage
-
-This plugin is made up of 2 packages:
-
-1. `@janus-idp/backstage-plugin-ocm-backend` connects backstage server to OCM.
-2. `@janus-idp/backstage-plugin-ocm` contains frontend components, requires `*-backend` to be present and set up.
-
-> If you are interested in Resource discovery only and you don't want to use any of our frontend components, you can install and configure only the `@janus-idp/backstage-plugin-ocm-backend` package.
-
-### Setup `@janus-idp/backstage-plugin-ocm-backend`
+## Prerequisites
 
-1. Install the backend plugin:
+- OCM is deployed and configured on a Kubernetes cluster.
+- [Kubernetes plugin for Backstage](https://backstage.io/docs/features/kubernetes/overview) is installed.
+- Kubernetes plugin is configured and can connect to the hub cluster for OCM using a `ServiceAccount`. The cluster is accessed using Backstage backend, therefore, the cluster requires a permanent connection with constant RBAC scope.
```

Comment:
I think we should clarify this. This is obsolete information. Kubernetes plugin for Backstage is not a prerequisite. It's optional. OCM plugin can either reference a cluster directly or defer to the Kubernetes plugin for details. We manage the connection ourselves.

Let's remove it from prerequisites and instead add a note below (in the config section) describing this optional behavior.

Reply:
Removed it and added in the config section.